backend/app/store/image/image.go

```diff
@@ -133,23 +149,28 @@ func (s *Service) ExtractPictures(commentHTML string) (ids []string, err error)
 	return ids, nil
 }
 
-// Cleanup runs periodic cleanup with TTL. Blocking loop, should be called inside of goroutine by consumer
+// Cleanup runs periodic cleanup with cleanupTTL. Blocking loop, should be called inside of goroutine by consumer
 func (s *Service) Cleanup(ctx context.Context) {
-	log.Printf("[INFO] start pictures cleanup, staging ttl=%v", s.TTL)
+	log.Printf("[INFO] start pictures cleanup, staging ttl=%v", s.cleanupTTL)
 
 	for {
 		select {
 		case <-ctx.Done():
 			log.Printf("[INFO] cleanup terminated, %v", ctx.Err())
 			return
-		case <-time.After(s.TTL / 2): // cleanup call on every 1/2 TTL
-			if err := s.store.Cleanup(ctx, s.TTL); err != nil {
+		case <-time.After(s.cleanupTTL):
+			if err := s.store.Cleanup(ctx, s.EditDuration); err != nil {
```

Comment:
the knowlege about internalas of other parts with such a concete EditDuration name looks strange. Not sure what should be the right name.Probably this is cleanupTTL, and the thing you currently call cleanupTTL is some sort of clanupInterval or cleanupDuration.

Reply:
Changed to `cleanupTTL` which is more in-line with original value.